tests/Unit/DocumentManagementTest.php

```
@@ -0,0 +1,167 @@
+            ->willReturn($expectedResponse);
+
+        // Act
+        $response = $appClient->listDocuments($filters, 0, 10);
+
+        // Assert
+        $this->assertIsArray($response);
+        $this->assertArrayHasKey('content', $response);
+        $this->assertArrayHasKey('totalElements', $response);
+        $this->assertEquals(2, $response['totalElements']);
+        $this->assertCount(2, $response['content']);
+    }
```

PHPMD (2.15.0)

The method it_lists_documents_with_filters is not named in camelCase. (undefined)

(CamelCaseMethodName)